### PR TITLE
Moar fantomas

### PR DESCRIPTION
**Pull request by nojaf**
## Description

Format all F# source code.

## How to test

CI should check it, otherwise `dotnet fantomas src tests samples --check`

## Related issues

https://github.com/giraffe-fsharp/Giraffe/issues/581

### CODE REVIEW

File: .editorconfig [hunk 1]
```diff
@@ -0,0 +1,9 @@
+root = true
+
+[*.{fs,fsi,fsx}]
+end_of_line = lf
+fsharp_multiline_bracket_style = aligned
```
Comment (nojaf) [inline .editorconfig:5]: Easier for source controle diff.

File: .editorconfig [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+fsharp_multi_line_lambda_closing_newline = true
```
Comment (nojaf) [inline .editorconfig:6]: Bit of a personal preference.

File: .editorconfig [hunk 3]
```diff
@@ -0,0 +1,9 @@
...
+
+[{tests,samples}/**/*.fs]
+fsharp_experimental_elmish = true
```
Comment (nojaf) [inline .editorconfig:9]: Formats expressions like `foo [ bar1 ; bar2 ]` nicer.

File: README.md [hunk 4]
```diff
@@ -235,25 +235,10 @@ Please be aware that Giraffe strictly aims to remain as light as possible while
 
 ### Code conventions
 
-When making changes please use existing code as a guideline for coding style and documentation. For example add spaces when creating tuples (`(a,b)` --> `(a, b)`), annotating variable types (`str:string` --> `str : string`) or other language constructs.
+We use [Fantomas](https://fsprojects.github.io/fantomas/docs/index.html) to ensure consistent code formatting across the project. Before committing, format your code by running the following command:
```
Comment (nojaf) [inline README.md:238]: I did not do any attempt to keep the rules. They were implemented inconsistent anyway and I feel like the default from the style guide suffice here.

**Review by TheAngryByrd — APPROVED** [review]

**Review by 64J0 — APPROVED** [review]
Great PR, it will be awesome to use fantomas for the whole codebase!

**Pull request merged**